drivers/iio/position/admt4000.c

```diff
@@ -0,0 +1,1572 @@
+// SPDX-License-Identifier: GPL-2.0
+/*
+ *
+ *
+ *
+ * Copyright 2025 Analog Devices Inc.
+ */
+#include <linux/spi/spi.h>
+
+#include <linux/regmap.h>
+#include <linux/regulator/consumer.h>
+#include <linux/bitfield.h>
+#include <linux/iio/iio.h>
+#include <linux/iio/sysfs.h>
+#include <linux/iio/types.h>
+#include <linux/bitops.h>
+#include <linux/unaligned.h>
+#include <linux/gpio/consumer.h>
+#include <linux/iio/buffer.h>
+#include <linux/iio/trigger.h>
+#include <linux/iio/triggered_buffer.h>
+#include <linux/iio/trigger_consumer.h>
+
+//adg
+#include <linux/of.h>
+#include <linux/of_device.h>
+
+/*ADMT4000 Registers*/
```

Comment:
```suggestion
/* ADMT4000 Registers */
```
Apply to other comments as well.